tests/test_binder_file_field.py

```diff
@@ -166,6 +166,25 @@ def test_get_unknown_extension(self):
 			'/zoo/{}/binder_picture/?h={}&content_type=&filename={}'.format(zoo.pk, UNKNOWN_TYPE_HASH, filename),
 		)
 
+	def test_get_direct(self):
+		filename = 'pic.jpg'
+		zoo = Zoo(name='Apenheul')
+		zoo.binder_picture_direct = ContentFile(JPG_CONTENT, name=filename)
+		zoo.save()
+
+		response = self.client.get('/zoo/{}/'.format(zoo.pk))
+		self.assertEqual(response.status_code, 200)
+		data = jsonloads(response.content)
+
+		# Remove once Django 3 lands with: https://docs.djangoproject.com/en/3.1/howto/custom-file-storage/#django.core.files.storage.get_alternative_name
+		zoo.refresh_from_db()
+		filename = basename(zoo.binder_picture_direct.name) # Without folders foo/bar/
+
+		self.assertEqual(
+			data['data']['binder_picture_direct'],
+			'/zoo/{}/binder_picture_direct/?h={}&content_type=image/jpeg&filename={}'.format(zoo.pk, JPG_HASH, filename),
```

Comment:
Can we check the `X-Accel-Redirect` of the response here directly? Since the requests do not go via nginx the header should just be left alone and added to the response I think? 
Basically validate: If `serve_directly` then header is present in response, otherwise not.